packages/clerk-js/src/core/resources/SignIn.ts

```diff
@@ -986,6 +1004,15 @@ class SignInFuture implements SignInFutureResource {
           identifier = await getOKXWalletIdentifier();
           generateSignature = generateSignatureWithOKXWallet;
           break;
+        case 'solana':
+          if (!params.walletName) {
+            throw new ClerkRuntimeError('walletName is required for solana web3 authentication', {
+              code: 'missing_wallet_name',
+            });
+          }
```

Comment:
i would expect for this check to exist within the utils methods, for example `getSolanaIdentifier()`, instead of requiring every caller to check it